src/GaussianProcessEmulator.jl

```diff
@@ -56,36 +57,33 @@ $(DocStringExtensions.FIELDS)
 
 """
 struct GaussianProcess{FT<:AbstractFloat, GPM}
-    "training inputs/parameters; N_samples x input_dim"
-    inputs::Array{FT}
-    "training data/targets; N_samples x output_dim"
-    data::Array{FT}
+    "training inputs and outputs, data stored in columns"
+    input_output_pairs::PairedDataContainer{FT}
     "mean of input; 1 x input_dim"
     input_mean::Array{FT}
     "the Gaussian Process (GP) Regression model(s) that are fitted to the given input-data pairs"
+    sqrt_inv_input_cov::Union{Nothing, Array{FT, 2}}
+    "prediction type (`y` to predict the data, `f` to predict the latent function)"
     models::Vector
     "the singular value decomposition of obs_noise_cov, such that obs_noise_cov = decomposition.U * Diagonal(decomposition.S) * decomposition.Vt."
     decomposition::Union{SVD, Nothing}
     "whether to fit GP models on normalized inputs ((inputs - input_mean) * sqrt_inv_input_cov)"
     normalized::Bool
     "square root of the inverse of the input covariance matrix; input_dim x input_dim"
-    sqrt_inv_input_cov::Union{Nothing, Array{FT, 2}}
-    "prediction type (`y` to predict the data, `f` to predict the latent function)"
     prediction_type::Union{Nothing, PredictionType}
 end
 
 
 """
-GaussianProcess(inputs::Array{FT, 2}, data::Array{FT, 2}, package::GPJL; GPkernel::Union{K, KPy, Nothing}=nothing, obs_noise_cov::Union{Array{FT, 2}, Nothing}=nothing, normalized::Bool=true, noise_learn::Bool=true, prediction_type::PredictionType=YType()) where {K<:Kernel, KPy<:PyObject}
+    GaussianProcess(inputs::Array{FT, 2}, data::Array{FT, 2}, package::GPJL; GPkernel::Union{K, KPy, Nothing}=nothing, obs_noise_cov::Union{Array{FT, 2}, Nothing}=nothing, normalized::Bool=true, noise_learn::Bool=true, prediction_type::PredictionType=YType()) where {K<:Kernel, KPy<:PyObject}
```

Comment:
Outdated description, does not contain `input_output_pairs::PairedDataContainer{FT},`